README.md

```diff
@@ -213,6 +213,26 @@ Generated code requires:
 Example with simple schema and few queries and mutations is available [here](https://github.com/mirumee/ariadne-codegen/blob/main/EXAMPLE.md).
 
 
+## Generating graphql schema's python representation
+
+Instead of generating client, you can generate file with graphql schema's python representation. To do this provide `graphqlschema` as an argument:
```

Comment:
```suggestion
Instead of generating client, you can generate file with a copy of GraphQL schema as `GraphQLSchema` declaration. To do this call `ariadne-codegen` with `graphqlschema` argument:
```